File: CC2_ComProg1/CC2_Finals/Looping/MultiplicationTable.java
/**
 * @author pao
 * BSCS 1B
 */
import java.util.Scanner;
public class MultiplicationTable {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter starting number : ");
        int start = sc.nextInt();
        System.out.print("Enter ending number   : ");
        int end = sc.nextInt();
        System.out.print("Enter table number    : ");
        int table = sc.nextInt();
        while (start<=end) {
            int temp = start * table;
            System.out.println(start + " * " + table + " = " + temp);
            start++;
        }
    }
}

File: Practice Codes/CC2 Java to Python/Finals/Looping/multiplication_table.py
print(f"""|--------------------------------|
|      Multiplication Table      |
|--------------------------------|""")
start = int(input("Enter starting number : "))
end = int(input("Enter ending number   : "))
table = int(input("Enter table number    : "))
while start <= end:
    temp = start * table
    print(f"{start} * {table} = {temp}")
    start += 1
